### PR TITLE
RIA-7433 Generate letter when LO changes hearing centre for detained…

### PR DESCRIPTION
**Pull request by neha-aggarw**
… internal appeals

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java [hunk 1]
```diff
@@ -64,6 +64,7 @@ public enum DocumentTag {
     INTERNAL_HEARING_REQUIREMENTS_UPDATED_LETTER("internalHearingRequirementsUpdatedLetter", CaseType.ASYLUM),
     MAINTAIN_CASE_UNLINK_APPEAL_LETTER("maintainCaseUnlinkAppealLetter", CaseType.ASYLUM),
     INTERNAL_UPLOAD_ADDITIONAL_EVIDENCE_LETTER("internalUploadAdditionalEvidenceLetter", CaseType.ASYLUM),
+    INTERNAL_CHANGE_HEARING_CENTRE_LETTER("internalChangeHearingCentreLetter", CaseType.ASYLUM),
```
Comment (hussain-faruk) [inline src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java:67]: dont think we need to specify internal here. A case will be internal or not.

**Comment by hussain-faruk** [comment]
we should add a FT for detained non-ada case. Even though we know it will work if the ada scenario works, a ft dedicated to detained non-ada ensures against future changes.

**Review by ikirsanov — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java (hunk 1 above)
Comment (neha-aggarw) [reply src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/DocumentTag.java:67]: But the letter we are generating is only for internal. Not for the other journey types.

**Review by ikirsanov — DISMISSED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Review by ikirsanov — APPROVED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Pull request merged**